recipe/meta.yaml

```
@@ -20,14 +22,18 @@ requirements:
     - meson
 
 test:
+  requires:
+    - chrpath  # [linux]
   commands:
     - test -f ${PREFIX}/lib/liborc-0.4${SHLIB_EXT}  # [unix]
+    - chrpath -l ${PREFIX}/lib/liborc-0.4.so      | grep -qF 'RPATH=$ORIGIN/.'  # [linux]
+    - chrpath -l ${PREFIX}/lib/liborc-0.4-test.so | grep -qF 'RPATH=$ORIGIN/.'  # [linux]
     - orcc --help
     - orc-bugreport --help
  
 about:
   home: https://gstreamer.freedesktop.org/modules/orc.html
-  license: Custom
+  license: BSD-2-Clause AND BSD-3-Clause
```

Please double-check this!

---

Here is the licensing:
https://gitlab.freedesktop.org/gstreamer/orc/blob/master/COPYING

I think it should be custom, isn't it?

---

The Cerbero build system (which is the official build system for GStreamer) identifies the orc license as BSD-like. See: https://gitlab.freedesktop.org/gstreamer/cerbero/blob/69708f677d1d558ce80d5e31d5622132795aa212/recipes/orc.recipe#L17

We could do something similar:
```suggestion
  license: BSD-like
  license_family: BSD
```

---

Thanks @kleisauke 

Would you like to be included as a maintainer of this recipe?

---

@sebastian-luna-valero Thanks for asking. I don't use Conda on a daily basis, so I'm probably not good at maintaining this recipe.

As an aside, thanks for maintaining the libvips recipe!

---

My point was that the license does not seem to be "custom", really, but just looks to be `BSD-2-Clause AND BSD-3-Clause`. That `LicenseA AND LicenseB` is a SPDX license expression (see https://spdx.org/ids-how for details) and we (as in parts of conda-forge and others in the Conda ecosystem) are slowly pushing to use those expressions for quite some time.

The only thing that is ambiguous about the license are the very first words: "The majority of ...". That wording is unfortunate because it is **not** clear if they mean "Apart from the exceptions mentioned below ..." with this or if the license is in fact `BSD-2-Clause AND BSD-3-Clause AND some-other-unnamed-licenses`.

---

Ignore my comment about the BSD-like change (it was due to the first words of the license). The `BSD-2-Clause AND BSD-3-Clause` SPDX license expression is more correct and less vague.

(I think we can (safely) interpret the first words as _"With the exception of source that contains the Mersenne Twister algorithm ..."_)